pkg/apis/logstash/v1alpha1/logstash_types_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+package v1alpha1
```

Comment:
Nit: additional empty line between license header and package declaration.
```suggestion

package v1alpha1
```